operators/config/operator/all-in-one/cluster_role.template.yaml

```diff
@@ -54,11 +54,10 @@ rules:
   resources:
   - elasticsearches
   - elasticsearches/status
+  - elasticsearches/finalizers
   - enterpriselicenses
   - enterpriselicenses/status
   - trustrelationships
-  - users
-  - users/status
```

Comment:
Do you have any idea how to prevent this in the future?

Reply:
Not really, it's not clean but harmless and we are not supposed to remove custom resources very often when we passe the GA release.